Content.Server/Mech/Equipment/EntitySystems/MechGrabberSystem.cs

```diff
@@ -182,6 +190,10 @@ private void OnMechGrab(EntityUid uid, MechGrabberComponent component, DoAfterEv
         if (!_mech.TryChangeEnergy(equipmentComponent.EquipmentOwner.Value, component.GrabEnergyDelta))
             return;
 
+        if (TryComp<StrapComponent>(args.Args.Target, out var strapComp) && strapComp.BuckledEntities != null) // Frontier: Remove people from chairs
+            foreach (var buckleUid in strapComp.BuckledEntities)
+                _buckle.TryUnbuckle(buckleUid, buckleUid, true);
```

Comment:
Honestly, just force unbuckle here.  If you just buckled in, the check'll fail (CanUnbuckle, SharedBuckleSystem.cs:L525)